docs/contributor/04-10-technical-design.md

```diff
@@ -2,162 +2,110 @@
+![Kyma Istio Operator Overview](../assets/istio-operator-overview.svg)
 
-In order to transition to a more modularised architecture, the [IstioOperator resource](https://github.com/kyma-project/kyma/tree/main/resources/istio),
-the [additional istio-resources](https://github.com/kyma-project/kyma/tree/main/resources/istio-resources), and
-the [certificates](https://github.com/kyma-project/kyma/tree/main/resources/certificates) must be moved to the new modules.
+## Istio CR
 
-#### Kyma Istio Operator resource
+The [Istio CR](../user/03-technical-reference/istio-custom-resource/01-30-istio-custom-resource.md) a namespace-scoped resource that is used to manage the Istio installation. The reason for Istio CR being namespace-scoped is that it was created and used before the Kyma Istio Operator was introduced.
+There is no advantage to the Istio CR being namespace-scoped as the Kyma Istio Operator only supports one Istio CR per cluster. However, it is not possible to change it to cluster-scoped without breaking changes.
 
-The Kyma Istio Operator resource is moved into the new Kyma Istio Operator. It is used to define default values for Istio, which the user can customise by modyfying Istio CR.
+If multiple Istio CRs exist on the cluster, the Kyma Istio operator will only use the oldest Istio CR for reconciliation and put the other Istio CRs in an error state.
 
-#### Istio resources
+## Istio version
 
-##### Istio Grafana dashboards
+The version of Istio is coupled to the version of the Kyma Istio Operator. This means that a particular version of the operator is responsible for a particular version of Istio and a new version of the operator is released to support a new version of Istio.
+Upgrading to a new version of the operator will automatically update Istio if the version of Istio has changed in the new operator version.
 
-It still needs to be decided who will have ownership of the dashboards. To make the right choice, such aspects as the change interval or relevance of Istio version updates should be considered.
+## Version upgrade
 
-##### Istio ServiceMonitor
+The Istio version upgrade is limited to supporting only one minor version (1.2.3 -> 1.3.0). This implies that this is also relevant when upgrading the Kyma Istio Operator to a new version.
+This means that when upgrading the operator to a new version, a minor version of the operator can only be skipped if the difference between the Istio versions in the new operator version is no more than one minor version.
 
-We do not consider Istio ServiceMonitor in the design, because this resource is planned to be replaced.
-For more information, see this [PR](https://github.com/kyma-project/kyma/pull/16247).
+If the difference between the current and the target version of Istio is greater than one minor version (1.2.3 -> 1.4.0), the reconciliation will fail.
+The same applies to major version upgrades (1.2.3 -> 2.0.0) and downgrades (1.2.3 -> 1.2.2).
```

Comment:
Downgrades are not supported

Reply:
Yes, that's what is mentioned in the docs. That in case of major upgrade or downgrade, the reconciliation will fail as well. Maybe it's not obvious enough and we should write it like that:
```suggestion
The reconciliation will also fail for major version upgrades (1.2.3 -> 2.0.0) and downgrades (1.2.3 -> 1.2.2), as these are also not supported.
```

Reply:
I just don't see if mentioning the path "(1.2.3 -> 1.2.2)" makes sense, as it doesn't apply to either patch or minor, it's a general behaviour. At least specifying that it applies to `any` downgrade maybe would be better.

Reply:
Ah Okay, now I get your point.
So we should rather have it like that?
```suggestion
The reconciliation will also fail for major version upgrades (1.2.3 -> 2.0.0) and version downgrades, as these are also not supported.
```

Reply:
👍